src/scripts/main.js

```diff
@@ -1,8 +1,27 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('table');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const myPeople = [...people].map(human => {
```

Comment:
```suggestion
const myPeople = people.map(human => {
```
map не мутирующий метод